tests/EventTests/EventBuilderTest.php

```diff
@@ -309,6 +309,100 @@ public function testCreateImpressionEventWithUserAgentWhenBotFilteringIsEnabled(
         $this->assertTrue($result[0], $result[1]);
     }
 
+    public function testCreateImpressionEventWithAttributesOfValidType()
```

Comment:
You could have edited any exisiting testcase with attributes. No need for a duplicate test case